wgpu-core/src/command/compute.rs

```diff
@@ -841,10 +864,6 @@ fn dispatch_indirect(
         .device
         .require_downlevel_flags(wgt::DownlevelFlags::INDIRECT_EXECUTION)?;
 
-    state
-        .scope
-        .buffers
-        .merge_single(&buffer, hal::BufferUses::INDIRECT)?;
```

Comment:
question: Why do we not call `merge_single` in the version of code with indirect validation? 🤔 Is that on purpose?

Reply:
@cwfitzgerald explained to me that we're using several different layers of usage tracking here, and that we needed a different path when issuing these encoder commands under the hood. 👍🏻 I think we're good to go here?